### PR TITLE
Script includes 2

### PR DESCRIPTION
**Pull request by michael-n-cooper**
final fixes

### CODE REVIEW

**Comment by michael-n-cooper** [comment]
This gets the example and note numbering going, I hope it's the right algorithm! 

- Notes and examples get sequential numbers within a section, blockquote, or dd
- If there is only one, no number
- Content included from WCAG is also numbered
- The separation of numbering between included content and section notes creates number jumps at times
- Some styling work is needed, I tried to leave semantics for it

**Review by daniel-montalvo — APPROVED** [review]

**Comment by maryjom** [comment]
There is a problem with the Numbering of the notes. Editor's notes are now numbered notes rather than Editor's notes. So there needs to be different handling depending on the different type of note.

File: wcag2ict.js [hunk 1]
```diff
@@ -102,6 +106,134 @@ function prepTerm(n) {
...
+	sectionsWithNotes.forEach(function(sec) {
+		if (sec.noteprocessed) return;
+		var notes = sec.querySelectorAll('.note');
+		// no notes, shouldn't happen
+		if (notes.length == 0) return;
+		// one note
+		if (notes.length == 1) {
+			// respec note, do nothing
+			// included note, add marker
+			if (notes[0].querySelector(".marker span") == null) addNoteMarker(notes[0], "Note: ");
```
Comment (maryjom) [inline wcag2ict.js:128]: Problem with algorithm. This is making Editor's notes numbered notes.

Comment (maryjom) [reply wcag2ict.js:128]: NVM...the algorithm is good. I found a place where I had used "note" and note "ednote" for the formatting so it was my bug in the document.

Comment (daniel-montalvo) [reply wcag2ict.js:133]: Hey @maryjom Thanks much for pointing this out. I've added some logic to discern which of these notes are not editor notes. I think this is better now.

Comment (maryjom) [reply wcag2ict.js:133]: @daniel-montalvo Yes, that change works. I also added a change to remove the "Guidance When Applying" sections from the TOC for the glossary terms sections. This also significantly cleans up the TOC. I managed to copy existing similar code and figure out what needed to change to get this to work.

**Review by maryjom — APPROVED** [review]
I think this is good enough scripting for this round. I do need to make some minor content updates to clean out extra text "Example:" because now the scripting is adding it and so there's double text.

**Pull request merged**